modules/lambda_teams_alert_monitor_log/README.md

```diff
@@ -36,3 +36,4 @@ module "monitor_log" {
 }
 
 ```
```

Comment:
The `base_infrastructure` is not a useful abstraction by itself. You should demote it to a sub-module, that is called by your modules, then have optional variables to pass in your own resources such as the `build_bucket`.

Having a "general" module fits your specific use-case but nothing else.

Reply:
Good point. Will remove this